documentation/bootstrap_upgrade_notes.md

```diff
@@ -0,0 +1,47 @@
+# Bootstrap upgrade notes
+
+**NOTE: If you have previously disabled your CA, please read this.**
+
+In the past, puppetserver users who have modified their `bootstrap.cfg` file to disable the CA on their compile masters and support a multi-master setup have been subject to a painful upgrade experience.
+
+Starting in puppetserver 2.5.0 this problem will start to be resolved. However, it's important to note that users who have disabled the CA service in bootstrap.cfg will need to exercise caution when upgrading to 2.5.0. Subsequent releases should no longer be subject to this issue. This page will walk you through what upgrading will be like if you are managing `bootstrap.cfg` yourself.
```

Comment:
> Subsequent releases should no longer be subject to this issue. 

Does this mean that users upgrading from Puppet Server 2.4 or earlier to Puppet Server 2.5.1 or newer won't need to create a `ca.cfg` file before upgrading?